src/NuGet.Core/NuGet.Frameworks/DefaultFrameworkMappings.cs

```diff
@@ -14,515 +14,489 @@ namespace NuGet.Frameworks
-                            CreateGenerationAndStandardMapping(
-                                FrameworkConstants.CommonFrameworks.Net45,
-                                FrameworkConstants.CommonFrameworks.DotNet52,
-                                FrameworkConstants.CommonFrameworks.NetStandard11),
-
-                            // net451 -> dotnet5.3, netstandard1.2
-                            CreateGenerationAndStandardMapping(
-                                FrameworkConstants.CommonFrameworks.Net451,
-                                FrameworkConstants.CommonFrameworks.DotNet53,
-                                FrameworkConstants.CommonFrameworks.NetStandard12),
-
-                            // net46 -> dotnet5.4, netstandard1.3
-                            CreateGenerationAndStandardMapping(
-                                FrameworkConstants.CommonFrameworks.Net46,
-                                FrameworkConstants.CommonFrameworks.DotNet54,
-                                FrameworkConstants.CommonFrameworks.NetStandard13),
-
-                            // net461 -> dotnet5.5, netstandard2.0
-                            CreateGenerationAndStandardMapping(
-                                FrameworkConstants.CommonFrameworks.Net461,
-                                FrameworkConstants.CommonFrameworks.DotNet55,
-                                FrameworkConstants.CommonFrameworks.NetStandard20),
-
-                            // net462 -> dotnet5.6, netstandard2.0
-                            CreateGenerationAndStandardMapping(
-                                FrameworkConstants.CommonFrameworks.Net462,
-                                FrameworkConstants.CommonFrameworks.DotNet56,
-                                FrameworkConstants.CommonFrameworks.NetStandard20),
-
-                            // netcore45 -> dotnet5.2, netstandard1.1
-                            CreateGenerationAndStandardMapping(
-                                FrameworkConstants.CommonFrameworks.NetCore45,
-                                FrameworkConstants.CommonFrameworks.DotNet52,
-                                FrameworkConstants.CommonFrameworks.NetStandard11),
-
-                            // netcore451 -> dotnet5.3, netstandard1.2
-                            CreateGenerationAndStandardMapping(
-                                FrameworkConstants.CommonFrameworks.NetCore451,
-                                FrameworkConstants.CommonFrameworks.DotNet53,
-                                FrameworkConstants.CommonFrameworks.NetStandard12),
-
-                            // xamarin frameworks
-                            CreateGenerationAndStandardMappingForAllVersions(
-                                FrameworkConstants.FrameworkIdentifiers.MonoAndroid,
-                                FrameworkConstants.CommonFrameworks.DotNet56,
-                                FrameworkConstants.CommonFrameworks.NetStandard21),
-
```

Comment:
why remove the extra lines? :) It improves readability.

Reply:
Good catch.  I didn't remove them, something must have formatted it that way.  I'll bring them back.